src/js/gutenberg-plugin.js

```diff
@@ -231,9 +424,19 @@ const ClassifAIPlugin = () => {
 			className="classifai-panel"
 		>
 			<>
-				<ClassifAIToggle />
-				<ClassifAIGenerateTagsButton />
+				{
+					userHasPermissions &&
+					isLanguageProcessingEnabled &&
+					isPosTypeSupported &&
+					isPostStatusSupported && (
+						<>
+							<ClassifAIToggle />
+							<ClassifAIGenerateTagsButton />
+						</>
+					)
+				}
 			</>
+			<ClassifAITSpeechSynthesisToggle audioId={ audioId } />
```

Comment:
So the issue with this new approach is this ClassifAI panel will always show, even if all features are turned off. Maybe that's okay? But it is a change to how we approach this right now

Reply:
@dkotter Yes, I have intentionally kept it this way.
Since we have multiple services under the same panel, the conditional check just to show/hide the panel will be way too lengthy and complicated. This is why I am always showing the Speech synthesis controlled but enabled/disabled based on supported post type.